### ISSUE TITLE
Problem about the stop gradient operation for negative pairs.

### ISSUE THREAD

**Issue by MosPicDev** [alexandonian/contrastive-feature-loss#3]
According to the equation (5) in paper, which says that "gradients are prevented from flowing through the negatives". Then for the PatchNCELoss function in https://github.com/alexandonian/contrastive-feature-loss/blob/788ac92d09d35a3f973a368fe8e545024e970eb4/models/networks/nce.py#L30, should we detach feat_k after the calculation of pos logits?
